core/src/main/resources/error/error-classes.json

```
@@ -290,6 +290,46 @@
           "Null typed values cannot be used as arguments of <functionName>."
         ]
       },
+      "NUMBER_FORMAT_CURRENCY_MUST_BEFORE_DECIMAL" : {
+        "message" : [
+          "Currency characters must appear before any decimal point in the number format: '<numberFormat>'"
+        ]
+      },
+      "NUMBER_FORMAT_CURRENCY_MUST_BEFORE_DIGIT" : {
+        "message" : [
+          "Currency characters must appear before digits in the number format: '<numberFormat>'"
+        ]
+      },
+      "NUMBER_FORMAT_EMPTY" : {
+        "message" : [
+          "The format string cannot be empty"
```

nit: but for consistency w/ other error messages. Could you add a dot at the end.
```suggestion
          "The format string cannot be empty."
```

---

Done